### ISSUE TITLE
Need an Index Toggler

### ISSUE THREAD

**Issue by h-zone** [davidpiesse/nova-toggle#11]
^^ subject
To toggle a single row we need to make additional clicks for now. this is bad to force clients making senseless actions.

**Comment by h-zone** [comment]
And so, this is a useful field for nova, saves time and nerves

**Comment by davidpiesse** [comment]
Hi @h-zone ,
How would you expect this to work?
Just need to understand a use case and expected UX

**Comment by atmediauk** [comment]
@davidpiesse This sounds cool. Not the OP but I think with regard to UX it would be cool if it worked like the Button Field plugin (https://novapackages.com/packages/dillingham/nova-button)

The toggle would be displayed for each entry, e.g if a record is enabled, it can then be toggled to disabled straight from the index.

**Comment by davidpiesse** [comment]
OK - so make the toggle visible on the Index view.
I am guessing smaller too?
:)
D

**Comment by atmediauk** [comment]
@davidpiesse Yes please, essentially as it is now but actually toggle-able on index.  I think smaller would work better as well. Thanks!

**Comment by davidpiesse** [comment]
I am currently working on the new version for the latest upcoming Nova version (v4) which will use Vue 3; and I will be implementing Headless UI's toggle. https://headlessui.dev/vue/switch

This might be something that makes it into the new version.
